src/main/scala/codacy/plugins/test/DockerHelpers.scala

```diff
@@ -2,36 +2,16 @@ package codacy.plugins.test
 
 import java.nio.file.{Files, Path}
 
-import codacy.TryExtension
-import codacy.docker.api._
-import codacy.plugins.docker.Pattern
-import play.api.libs.json.Json
-import plugins._
-
 import _root_.scala.sys.process._
-import scala.util.Try
 
 object DockerHelpers {
 
   val dockerRunCmd = List("docker", "run", "--net=none", "--privileged=false", "--user=docker")
 
-  def toPatterns(toolSpec: Tool.Specification): Seq[Pattern] = toolSpec.patterns.map { case patternSpec =>
-    val parameters = patternSpec.parameters.map(_.map { param =>
-      (param.name.value, Json.toJson(param.default))
-    }.toMap)
-
-    Pattern(patternSpec.patternId.value, parameters)
-  }.toSeq
-
-  def toPatterns(patterns: Seq[PatternSimple]): Seq[Pattern] = patterns.map {
-    case pattern =>
-      Pattern(pattern.name, pattern.parameters)
-  }
-
-  def testFoldersInDocker(dockerImageName: DockerImageName): Seq[Path] = {
+  def testFoldersInDocker(dockerImageName: String): Seq[Path] = {
```

Comment:
Do we have to lose the `DockerImageName` type as input?

Reply:
I deleted DockerImageName when the ```class DockerPlugin(val dockerImageName: DockerImageName)``` was deleted because it was dealing with running the docker commands directly. 

I created ```final case class DockerImage(name: String, version: String)``` to have both name and version separated and I thought about creating two more case classes for name and version but I don't know if it is worth it...

Reply:
Forget it, now I understand. It's done, thanks! :+1: